Shared/Components/TruncatedText.swift

```diff
@@ -31,7 +31,7 @@ struct TruncatedText: View {
 
     private var isTruncatedBinding: Binding<Bool>
     private var onSeeMore: () -> Void
-    private let seeMoreText = "\u{2026} See More"
+    private let seeMoreText = L10n.ellipsisSeeMore
```

Comment:
For now, I would ask we don't localize the strings with an ellipsis explicitly. This is because I will probably remove the ellipsis from these strings and design altogether and trying to get it accurately represented in the strings file and ensuring it's localized in all languages is just more work. I also want to minimize strings as possible.

So, for these just concatenate them: `"\u{2026}" + L10n.ellipsisSeeMore`